core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -788,7 +791,11 @@ public void clone(String srcTableName, String newTableName, CloneConfiguration c
   @Override
   public void rename(String oldTableName, String newTableName) throws AccumuloSecurityException,
       TableNotFoundException, AccumuloException, TableExistsException {
-
+    checkArgument(oldTableName.length() <= MAX_TABLE_NAME_LEN,
```

Comment:
Why does the old table need to be below limit?

Reply:
@keith-turner Again, I might be misunderstanding ctubbsii's feedback on this one. I thought the intention in the new version was to essentially force user's to rename tables to something that conforms to the 1024 char limit.

Reply:
It seems like if the user had created a table name of len 1234 in ver 2.0.0 that with this check they would never be able to rename that table in version 2.1.0.

Reply:
@keith-turner Ok. I feel foolish now. You are correct. My goal was to generate a warning message to the user. In doing so, I am preventing the operation from even occuring! I will remove the old name check for renaming namespaces and tables. Also, I will generate the warning message without using checkArgument.

Reply:
>  I will generate the warning message without using checkArgument

Are you thinking of a warning for the case where the old name is too long?  Personally I would skip that.

Reply:
> Are you thinking of a warning for the case where the old name is too long? Personally I would skip that.

For rename, I agree. There's no point in warning when the user is currently performing an action that makes the problem go away.

Reply:
@keith-turner @ctubbsii Thanks for being patient with me, guys. I think I have addressed the current issues with my latest commits.

Reply:
No problem! I'll try to take a look soon.

Reply:
@cradal did you see the suggestions I made to replace 1024 with constants in some error message strings?

Reply:
@keith-turner Yessir. I rolled them into my commits.

Reply:
@cradal  when I look at the diffs on github I still see some error messages with 1024 in them.

Reply:
@keith-turner Apologies. I thought we caught them all with the peer review and commits last week. I went through the relevant classes and did a string search for 1024. I think we finally got them all with my latest commit.